site/src/comparison.rs

```diff
@@ -193,9 +194,9 @@ impl ComparisonSummary {
             .cloned()
             .collect::<Vec<_>>();
         // Skip empty commits, sometimes happens if there's a compiler bug or so.
-        if comparisons.len() == 0 {
-            return None;
-        }
+        // if comparisons.len() == 0 {
+        //     return None;
+        // }
```

Comment:
I think this needs to be reverted.

Reply:
Then the summary table would need to take `Option<ComparisonSummary>` instead of `ComparisonSummary`, which would make the code a bit more complicated. It seems better to me to handle the case "no data available" without any special case.

But we can also solve it by passing an empty summary to the table function if this function returned None, we just need to add a constructor that creates an empty summary.

Reply:
Ok that's fine. Then it just needs to be deleted (instead of commented out).